Source/Scene/PointCloud3DTileContent.js

```diff
@@ -781,8 +785,11 @@ define([
         var hasNormals = content._hasNormals;
         var hasBatchIds = content._hasBatchIds;
         var backFaceCulling = content._backFaceCulling;
+        var normalShading = content._normalShading;
         var vertexArray = content._drawCommand.vertexArray;
 
+        var normalsEnabled = hasNormals && normalShading;
```

Comment:
I made those edits, although I'm don't totally understand the relationship between backface culling and normal shading in that function